packages/web-components/src/components/content-section/content-section-heading.ts

```diff
@@ -23,6 +23,13 @@ class DDSContentSectionHeading extends LitElement {
   @property({ reflect: true })
   slot = 'heading';
 
+  connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'heading');
+    }
```

Comment:
Could you add `aria-level`, too? Here's an [example](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.15.0-rc.0/packages/web-components/src/components/card/card-heading.ts#L32-L34) for one representing `<h3>`.